gulpfile.js

```diff
@@ -118,6 +128,34 @@ gulp.task('build-ci', ['clean'], function (cb) {
     runSequence('build', cb);
 });
 
+gulp.task('build-test', function buildTest(cb) {
+    var config = _.defaultsDeep({}, require('./webpack.config'));
+
+    config.entry = {};
+    // the key in the entry, formed by path.join, determines the target folder structure of the test file
+    config.entry[path.join('request', 'repository', 'request-repository-cache')] = './test/request/repository/request-repository-cache.spec.ts';
+
+    config.output.path = testSettings.output;
+
+    var started = false;
+    function processResult(err, stats) {
+        gutil.log('Webpack\n' + stats.toString(config.log));
+
+        if (!started) {
+            started = true;
+            cb();
+        }
+    }
+
+    var compiler = webpack(config);
+    compiler.run(processResult);
+});
```

Comment:
When we get to the point where we need to mock modules, I'm planning to use something like mockery.

Regarding manually connecting tests, it will be changed by my next PR.